src/main/java/past/cwdt/bookstore/dto/BookDto.java

```diff
@@ -0,0 +1,15 @@
+package past.cwdt.bookstore.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class BookDto {
```

Comment:
Use record as you are using Java 17. 
To avoid errors update checkstyle plugin version to 3.3.0